<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Materia Formativa</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="edit_nombre_formativa">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_requiere_formativa">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_nombre_formativa">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_creditos_formativa">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_periodo_formativa">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_periodo_formativa">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Periodo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_horario_formativa">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Horario:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_hteoria_formativa">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_hteoria_formativa">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Horas teoria:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_htrabajo_formativa">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_htrabajo_formativa">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Horas trabajo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_requiere_formativa">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Requiere:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_creditos_formativa">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Créditos:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_area_formativa">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_mensaje_formativa">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Mensaje</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_cursada_formativa">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_horario_formativa">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_hlaboratorio_formativa">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Horas laboratorio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cursada_formativa">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Es cursada por:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_agregar_formativa">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_eliminar_formativa">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>420</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ensenada_formativa">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Es enseñada por:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_hlaboratorio_formativa">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_ensenada_formativa">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_area_formativa">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Área:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_optativa">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Optativa</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>519</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
